xla/tests/onednn_layer_norm_test.cc

```diff
@@ -141,10 +148,130 @@ TEST_F(LayerNormTest, LayerNormTest0_F16) {
   MatchOptimizedHlo(layer_norm_module_str, onednn_layer_norm_);
 }
 
+TEST_F(LayerNormTest, LayerNormTest1_F16) {
```

Comment:
Could you please add a description above the test about what it tests for? E.g., how what it tests is different LayerNormTest0_F16, etc.

Reply:
While executing the ViT model in F16 we observed an accuracy issue which was due to the small epsilon value being constant folded. This PR is addressing that issue by adding a default value for epsilon and the test here is testing that scenario by not having an epsilon as opposed to the test above which contains an epsilon value.